components/forms/login/LoginForm.tsx

```diff
@@ -26,19 +28,19 @@ export default function LoginForm() {
     })
     if (!response.ok) {
       const status = response.status;
-      setError((status >= 400 && status < 500) ? "Incorrect username or password" : "Something went wrong, try again later")
+      setError((status === 401 || status === 403) ? t("incorrect") : t("error"))
```

Comment:
Can we make these a little more verbose?

```suggestion
      setError((status === 401 || status === 403) ? t("incorrect username or password") : t("something went wrong"))
```